openstack/identity/v3/extensions/trust/request_test.go

```diff
@@ -87,12 +66,3 @@ func TestTrustIDTokenID(t *testing.T) {
 
 	`)
 }
-
-func TestFailurePassword(t *testing.T) {
```

Comment:
This one never worked (those tests actually never ran before because of the filename), and doesn't seem really useful. I fixed the other one which was failing too. Be my guest if you understand and know how to fix this one.